src/Arcus.Messaging.Health/Arcus.Messaging.Health.csproj

```diff
@@ -28,7 +28,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="[8.0.13,10.0.0)" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    
+    <!-- TODO: will be removed in v3.0 -->
+    <PackageReference Include="System.Text.Json" Version="[8.0.5,10.0.0)" Condition="'$(TargetFramework)' == 'netstandard2.1'" />
```

Comment:
same remark here on the version-range.
Also wondering why we check here if the $TargetFramework is netstandard, while in the other csproj file we check if the TargetFramework is not .net 8

Reply:
Ah yes, that we need to streamline.